bom/public/pom.xml

```diff
@@ -15,6 +15,8 @@
     <packaging>pom</packaging>
 
     <properties>
+        <!-- This is a publicly distributed module that should be published: -->
+        <skipNexusStagingDeployMojo>false</skipNexusStagingDeployMojo>
```

Comment:
Come to think of it, this means we'll still publish this POM, even though it's not used in our other published POMs... right?

Do you think there's a way to skip publishing this particular POM?